content/changelog/2025/12-23-metabase-set-domain.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Set Metabase domain at creation
+description: You can now set a custom domain for Metabase admin interface when creating the add-on on Clever Cloud
+date: 2025-12-23
+tags:
+  - addons
+  - metabase
+authors:
+  - name: Sébastien Allemand
+    link: https://github.com/allemas
+    image: https://github.com/allemas.png?size=40
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+excludeSearch: true
+---
+
+When you deploy a Metabase add-on on Clever Cloud, you can access its admin interface through a `<random_chars>-metabase.services.clever-cloud.com` domain. You can now set a custom domain at creation through the `access-domain` option in Clever Tools:
```

Comment:
there no admin interface in metabase :D I don't know what we could put in its place